<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignDetectorAppClass</class>
 <widget class="QMainWindow" name="SignDetectorAppClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>873</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>564</width>
    <height>527</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SignDetectorApp</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Opcions</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Fitxer a executar</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Tipus de fitxer:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="videoImage">
              <item>
               <property name="text">
                <string>Video</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Imatge</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Path del fitxer a executar</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="videoPath"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Detector</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="ROILayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Mètode de ROI:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="typeROISelector">
              <item>
               <property name="text">
                <string>1. Tota la imatge</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2. Part central dreta de la imatge</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>3. Utilitzant detecció de linees mitjançant Localització 1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>4. Utilitzant detecció de linees mitjançant Hough</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Classificador</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Path de la base de dades de senyals:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="datasetPath"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="generaClassificador">
            <property name="text">
             <string>Genera una nova xarxa neuronal</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Executa detector</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pauseButton">
        <property name="text">
         <string>Acaba execució</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QOpenGLWCustom" name="openGLWidget">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>400</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTTerminal" name="text2show">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>704</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QTTerminal</class>
   <extends>QLabel</extends>
   <header>QTTerminal.h</header>
  </customwidget>
  <customwidget>
   <class>QOpenGLWCustom</class>
   <extends>QOpenGLWidget</extends>
   <header>QOpenGLWCustom.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="signdetectorapp.qrc"/>
 </resources>
 <connections/>
</ui>
